src/index.js

```diff
@@ -42,7 +43,11 @@ async function createTestCafe (hostname, port1, port2) {
         getValidPort(port2)
     ]);
 
-    return new TestCafe(hostname, port1, port2);
+    var testcafe = new TestCafe(hostname, port1, port2);
+
+    exitHook(cb => testcafe.close().then(cb));
```

Comment:
Maybe we should add some message to the log when exitHook called? Something like `Stopping TestCafe` or `Test run stopping`?
Otherwise now if you press `ctrl+c` in the terminal, it has no any response for some time (while testcafe is stopping) and you don't understand why the process is not aborted right now.
/cc @AndreyBelym @VasilyStrelyaev